wp-content/themes/core/assets/js/admin/editor/block-styles.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @function unregisterStyles
+ * @description Unregisters core block styles added client-side. Styles added
+ * 				server-side using register_block_style should be unregistered
+ * 				using the server-side unregister_block_styles function.
+ */
+
+const unregisterStyles = () => {
+	wp.blocks.unregisterBlockStyle( 'core/separator', 'default' );
+	wp.blocks.unregisterBlockStyle( 'core/separator', 'wide' );
+	wp.blocks.unregisterBlockStyle( 'core/separator', 'dots' );
+	wp.blocks.unregisterBlockStyle( 'core/quote', 'default' );
+	wp.blocks.unregisterBlockStyle( 'core/quote', 'plain' );
+};
```

Comment:
Should we move these to individual theme/blocks/core/*  blocks/component partials rather than here? I'm hoping we can keep all the scripts/styles related to a specific block in the block's directory.

Reply:
I considered this and I'm not opposed. The reason I didn't implement was because we don't yet have directories for the separator or quote block, and I didn't want to create them just to unregister the block style. I looked through a few block editor projects, which block styles are commonly unregistered and believe this would be a common occurrence- needing to create the full directory structure for core blocks just for the purpose of unregistering a style.

I agree it'd be nice to have all styles/scripts related to their respective block directory so if you feel strongly we should implement that now, I can do that.

Reply:
I'd say let's do it. It keeps the code consistent, demonstrates the approach we want to take, and makes adding additional settings, code, etc to those initial core blocks quicker on. a project.